### PR TITLE
Changed up the overlay on the episode recall overlay screen

### LINKED ISSUE #39: Clarify that the episodes being recalled are from day 1
When we are showing the episode overlay during episode recall, emphasize that the episode is from day 1.

### PR DESCRIPTION
**Pull request by jrosen081**
Closes: #39

### CODE REVIEW

File: screens/CameraRecording.tsx [hunk 1]
```diff
@@ -140,10 +142,21 @@ const CameraRecording: FunctionComponent<CameraRecordingProps> = ({
...
+            backgroundColor: overlayBackgroundColor,
+          }}
+        >
           {overlay}
           <Text
-            style={Styles.beginText}
+            style={{
+              ...Styles.beginText,
+              color:
+                overlayBackgroundColor === "rgba(0, 70, 67, .93)"
```
Comment (dfarooq610) [inline screens/CameraRecording.tsx]: Throw this in as a constant?

File: navigation/MainNavigator.tsx [hunk 2]
```diff
@@ -54,11 +56,12 @@ const MainNavigator: FunctionComponent = () => {
       return (
         <CameraFlowNavigator
           objects={navigationState.episodes}
+          overlayBackgroundColor="rgba(0, 70, 67, .93)"
```
Comment (dfarooq610) [inline navigation/MainNavigator.tsx]: Constant?

**Review by dfarooq610 — APPROVED** [review]
Looks **GREAT** to me

File: screens/CameraRecording.tsx (hunk 1 above)
Comment (jrosen081) [reply screens/CameraRecording.tsx]: That would have been way too smart for me. Will do

**Review by zefwang — APPROVED** [review]
lets go to (the) moon

**Pull request merged**